packages/openapi/src/middleware.ts

```diff
@@ -2,39 +2,76 @@ import { OpenAPI, RequestOptions, isValidationError } from './'
 
 import Koa from 'koa'
 
+interface ValidationOptions {
+  validateRequest?: boolean
+  validateResponse?: boolean
+}
+
 export function createValidatorMiddleware<T extends Koa.DefaultContext>(
   spec: OpenAPI,
-  options: RequestOptions
+  options: RequestOptions,
+  validationOptions: ValidationOptions | undefined = {
+    validateRequest: true,
+    validateResponse: true
+  }
 ): (ctx: Koa.Context, next: () => Promise<unknown>) => Promise<void> {
-  const validateRequest = spec.createRequestValidator<T['request']>(options)
-  const validateResponse =
-    process.env.NODE_ENV !== 'production' &&
-    spec.createResponseValidator(options)
```

Comment:
I feel like directly using `process.env.NODE_ENV` is not a fun time.
Will update Rafiki to have `validationOptions.validateReponse: process.env.NODE_ENV !== 'production'` instead